spec.html

```diff
@@ -30893,7 +30893,7 @@
           1. Assert: Type(_length_) is not Object.
           1. If NewTarget is *undefined*, throw a *TypeError* exception.
           1. If _length_ is *undefined*, throw a *TypeError* exception.
-          1. Let _numberLength_ be ToNumber(_length_).
+          1. Let _numberLength_ be ? ToNumber(_length_).
```

Comment:
Abrupt completions for `ToNumber` are checked by `?ToLength`, so having both, `?ToNumber` and `?ToLength`, is redundant.